sql/mob_droplist.sql

```diff
@@ -17438,6 +17438,7 @@ INSERT INTO `mob_droplist` VALUES (3223,0,0,1000,16017,@COMMON);
 INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,2811,@VCOMMON);
 INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,2815,@COMMON);
 INSERT INTO `mob_droplist` VALUES (3224,0,0,1000,2819,@VCOMMON);
+INSERT INTO `mob_droplist` VALUES (3225,0,0,1000,11343,@COMMON); --(Picolaton) Thrakon Breastplate
```

Comment:
sql comments need a space after the dashes or it doesn't import proper on all platforms. 
`--`  vs `-- `